engine/language-server/src/main/scala/org/enso/languageserver/runtime/ApiEventsMonitor.scala

```diff
@@ -7,16 +7,28 @@ import org.enso.polyglot.runtime.Runtime.Api
 import java.nio.charset.StandardCharsets
 import java.nio.file.{Files, Path, StandardOpenOption}
 import java.time.Clock
+import java.util.logging.XMLFormatter
 
 import scala.util.Try
+import java.util.logging.LogRecord
+import java.util.logging.Level
 
 /** Gather messages between the language server and the runtime and write them
-  * to the provided file in CSV format.
+  * to the provided file in XML format.
   *
   * @param path the path where to write the events
   * @param clock the system clock
   */
 final class ApiEventsMonitor(path: Path, clock: Clock) extends EventsMonitor {
+  private lazy val fmt = {
+    Files.write(
+      path,
+      "<?xml version='1.0'?>\n<records>".getBytes(StandardCharsets.UTF_8),
```

Comment:
@4e6, is it OK to just switch the format to XML? Or do we rely on the original format somewhere?

Reply:
I think the CSV output can be used by IDE profiling. Maybe we should ask @kazcw or @MichaelMauderer

Reply:
@kazcw, @MichaelMauderer can you take a look? I am not aware of any (finished) attempt to merge the IDE and engine logging together, but I may not have the overall picture. CCing @xvcgreg...